packages/actor-expression-evaluator-factory-base/lib/ActorExpressionEvaluatorFactoryBase.ts

```diff
@@ -0,0 +1,131 @@
+import type {
+  IBindingsAggregator,
+  MediatorBindingsAggregatorFactory,
+} from '@comunica/bus-bindings-aggeregator-factory';
+import type {
+  IActionExpressionEvaluatorFactory,
+  IActorExpressionEvaluatorFactoryArgs,
+  IActorExpressionEvaluatorFactoryOutput,
+} from '@comunica/bus-expression-evaluator-factory';
+import { ActorExpressionEvaluatorFactory } from '@comunica/bus-expression-evaluator-factory';
+import type {
+  MediatorFunctions,
+  IActionFunctions,
+  IActorFunctionsOutput,
+  IActorFunctionsOutputTerm,
+} from '@comunica/bus-functions';
+import type { MediatorQueryOperation } from '@comunica/bus-query-operation';
+import type { MediatorTermComparatorFactory, ITermComparator } from '@comunica/bus-term-comparator-factory';
+import { KeysExpressionEvaluator, KeysInitQuery } from '@comunica/context-entries';
+import type { IAction, IActorTest } from '@comunica/core';
+import { extractTimeZone } from '@comunica/expression-evaluator/lib/util/DateTimeHelpers';
+import type {
+  AsyncExtensionFunction,
+  IActionContext,
+
+} from '@comunica/types';
+import type * as RDF from '@rdfjs/types';
+import { LRUCache } from 'lru-cache';
+import type { Algebra as Alg } from 'sparqlalgebrajs';
+import { AlgebraTransformer } from './AlgebraTransformer';
+import { ExpressionEvaluator } from './ExpressionEvaluator';
+
+export function prepareEvaluatorActionContext(orgContext: IActionContext): IActionContext {
+  let context = orgContext;
+
+  context =
+    context.set(KeysExpressionEvaluator.now, context.get(KeysInitQuery.queryTimestamp) || new Date(Date.now()));
+
+  context = context.set(KeysExpressionEvaluator.baseIRI, context.get(KeysInitQuery.baseIRI));
```

Comment:
Good question. I think this replaces the getBaseExpressionContext of ActorQueryOperation.ts: https://github.com/comunica/comunica/blob/master/packages/bus-query-operation/lib/ActorQueryOperation.ts#L104-L124 . Do you suggest something else? 
(I did now remove ` context = context.set(KeysExpressionEvaluator.baseIRI, context.get(KeysInitQuery.baseIRI));`).

Reply:
Oh! I understand what you mean now. SPARQLEE used to have their own context items for this. It might be better to use the original context keys like `KeysInitQuery.baseIRI` and ` KeysInitQuery.queryTimestamp`?

Reply:
Indeed, if the semantics of the context entries remain identical (which should be possible for all/most?), then I'd reuse the same key.

Reply:
I was required to replace it with:
```
context = context.setDefault(KeysInitQuery.queryTimestamp, new Date(Date.now()));
context = context.setDefault(KeysInitQuery.functionArgumentsCache, {});
```
 is this oke?

Reply:
They were no needed